CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 Change log
 ==========
 
+1.5.0 (2015-10-13)
+------------------
+
+Major Features
+
+-   Compose is now available on windows.
+-   Environment variable can be used in the compose file. See
+    https://github.com/docker/compose/blob/master/docs/yml.md#variable-substitution
```

Comment:
Normally I'd like to link to a tag, but we don't create the tag until after the official release, so anyone reading these as part of an rc won't get a correct link.

I also can't link to a branch because we'll drop the bump-1.5.0 branch after release.

I guess that only leaves a commit, which is an ugly URL, but not sure what else to do